packages/visualizations/src/components/Table/Cell/Cell.svelte

```diff
@@ -21,6 +22,10 @@
         padding: var(--spacing-75);
     }
 
+    :global(.ods-dataviz--default td.rtl-direction) {
```

Comment:
From my tries, if we have `<table dir="rtl">` then text in td will be rtl. We might want to force ltr for some formats though (number, maybe URL).